pages/common/git-count-objects.md

```diff
@@ -0,0 +1,20 @@
+# git count-objects
+
+> Count unpacked number of objects and their disk consumption.
+> More information: <https://git-scm.com/docs/git-count-objects>.
+
+- Display count of all objects and their disk usage:
+
+`git count-objects`
+
+- Display count of all objects and their disk usage, displaying sizes in human readable units:
```

Comment:
Perhaps this will read better?
```suggestion
- Display a count of all objects and their total disk usage, displaying sizes in human readable units:
```